lib/DebugInfo/Symbolize/BugsnagSymbolize.cpp

```diff
@@ -68,19 +72,23 @@ SymbolizeResults BugsnagSymbolize(const char* filePath, bool includeInline, int6
   std::vector<SymbolizeResult> results;
 
   for (int i = 0; i < addressCount; i++) {
-    auto ResOrErr = Symbolizer.symbolizeCode(moduleName, addresses[i]);
+    std::string stringAddress = addresses[i];
```

Comment:
Consider naming this `hexAddress` so that we're not reflecting the type of the variable in its name